app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -343,6 +453,9 @@ const JenkinsSetup = (): JSX.Element => {
 
 	const isFetchingData = !serverName || !webhookUrl || !secret;
 
+	const enableFinishButton = (copyButtonStates.isCopyAdminSecret && copyButtonStates.isCopyAdminSiteName &&
```

Comment:
We should rename this to `allButtonsCopied/Clicked` or something descriptive of what the conditional is checking